aea/test_tools/test_cases.py

```diff
@@ -933,6 +941,31 @@ def teardown_class(cls) -> None:
         cls.agent_name = ""
 
 
+class AEATestCaseEmptyFlaky(AEATestCaseEmpty):
```

Comment:
Strictly speaking, one could also just use `AEATestCaseEmpty`. But the setup and teardown of class is called for each run. So it's better not to have multiple test methods to avoid unexpected behaviours.